app/auth/apikey_crud.py

```diff
@@ -159,7 +164,11 @@ def renew_key(
             conn.execute(
                 t.update()
                 .where(t.c.api_key == self.__hash(api_key))
-                .values(is_active=True, expiration_date=parsed_expiration_date)
+                .values(
```

Comment:
Since the July 16, the latest-sync-date is updated. However, the groups have not been resynced with Keycloak. To be corrected.